crates/kitsune_p2p/types/src/config.rs

```diff
@@ -119,6 +119,15 @@ pub mod tuning_params_struct {
         /// Should gossip dynamically resize storage arcs?
         gossip_dynamic_arcs: bool = false,
 
+        /// Allow only the first agent to join the space to
+        /// have a sized storage arc. [Default: false]
+        // This is an experimental feature that sets the first
+        // agent to join as the full arc and all other later
+        // agents to empty.
+        // It should not be used in production unless you understand
+        // what you are doing.
```

Comment:
This is worth being in the docs

```suggestion
        /// This is an experimental feature that sets the first
        /// agent to join as the full arc and all other later
        /// agents to empty.
        /// It should not be used in production unless you understand
        /// what you are doing.
```